src/fe-common/irc/module-formats.c

```diff
@@ -89,6 +89,9 @@ FORMAT_REC fecommon_irc_formats[] = {
 	{ "invitelist_long", "{channel $0}: invite {ban $1} {comment by {nick $2}, $3 secs ago}", 4, { 0, 0, 0, 1 } },
 	{ "no_such_channel", "{channel $0}: No such channel", 1, { 0 } },
 	{ "channel_synced", "Join to {channel $0} was synced in {hilight $1} secs", 2, { 0, 2 } },
+	{ "irctxt_help_start", "$1", 2, { 0, 0 } },
+	{ "irctxt_help_txt", "$1", 2, { 0, 0 } },
+	{ "irctxt_end_of_help", "$1", 2, { 0, 0 } },
```

Comment:
no need for irctxt_ prefix here, only for the constants. although I think I would rename it to server_help_start /IRCTXT_SERVER_HELP_START so it won't be confused with client-internal help